### PR TITLE
feat: thorchain LP flag and route

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Does what it says on the box

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

- contributes to https://github.com/shapeshift/web/issues/5939

## Risk

None

## Testing

- With the LP flag on, the Pools Page is displayed and routing is happy

### Engineering

- ☝🏽 

## Screenshots (if applicable)

- Flag off

Screenshot 2024-01-09 at 18 08 48

- Flag on

image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* `develop`
  * **PR #5966** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5966?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5966?utm_source=stack-comment).

File: src/Routes/RoutesCommon.tsx [hunk 1]
```diff
@@ -56,6 +57,16 @@ export const routes: NestedRoute[] = [
     mobileNav: false,
     disable: !getConfig().REACT_APP_FEATURE_THORCHAIN_LENDING,
   },
+  {
+    path: '/pools',
+    label: 'navBar.pools',
+    icon: <RiExchangeFundsLine />,
+    main: PoolsPage,
+    category: RouteCategory.Featured,
+    priority: 4,
```
Comment (kaladinlight) [inline src/Routes/RoutesCommon.tsx:66]: what are these priorities for? seeing several using `4`, just curious

**Review by kaladinlight — APPROVED** [review]

File: src/Routes/RoutesCommon.tsx (hunk 1 above)
Comment (gomesalexandre) [reply src/Routes/RoutesCommon.tsx:66]: Seems to be used for mobile sorting of nav routes only, summoning @reallybeard on whether or not we want to change it

**Review by 0xApotheosis — APPROVED** [review]
Does indeed what it says on the box 👌

File: src/Routes/RoutesCommon.tsx (hunk 1 above)
Comment (0xApotheosis) [reply src/Routes/RoutesCommon.tsx:66]: I'll merge this as is, we can fix the `priority` if @reallybeard thinks it's needed 🙏

**Pull request merged**